### PR TITLE
Added mail notify to the admins on delete policies

### PR DESCRIPTION
**Pull request by athiruma**
enable **dry_run=no** with all policies 
- ec2_idle
- ec2_stop
- ebs_unattached
- zombie_elastic_ips
- zombie_snapshots
- zombie_nat_gateways
- empty_buckets
- empty_roles
### Fixed the mail message

### CODE REVIEW

**Review by ebattat — APPROVED** [review]
/LGTM

File: jenkins/clouds/aws/policies/run_policies.py [hunk 1]
```diff
@@ -68,7 +68,10 @@ def get_custodian_policies(type: str = None):
...
             os.system(f"""podman run --rm --name cloud-governance -e account="PERF-SCALE" -e policy="{policy}" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE}" -e AWS_DEFAULT_REGION="{region}" -e dry_run="yes" -e LDAP_HOST_NAME="{LDAP_HOST_NAME}" -e special_user_mails={special_user_mails} -e account_admin="{account_admin}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}" -e policy_output="s3://{BUCKET_PERF_SCALE}/{LOGS}/{region}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
         # don't want to run it on PSAP
-        elif policy in ('ec2_stop', 'ec2_idle', 'zombie_elastic_ips', 'zombie_nat_gateways', 'zombie_snapshots'):
+        elif policy in ('ec2_idle', 'zombie_nat_gateways', 'zombie_snapshots'):
```
Comment (ebattat) [inline jenkins/clouds/aws/policies/run_policies.py:71]: where did you placed empty bucket and role ?

File: jenkins/clouds/aws/policies/run_policies.py [hunk 2]
```diff
@@ -84,9 +87,9 @@ def get_custodian_policies(type: str = None):
...
-    os.system(f"""podman run --rm --name cloud-governance -e account="PERF-DEPT" -e policy="{policy}" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF}" -e AWS_DEFAULT_REGION="{region}" -e dry_run="yes" -e LDAP_HOST_NAME="{LDAP_HOST_NAME}" -e special_user_mails={special_user_mails} -e account_admin="{account_admin}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}" -e policy_output="s3://{BUCKET_PERF}/{LOGS}/{region}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
-    os.system(f"""podman run --rm --name cloud-governance -e account="PERF-SCALE" -e policy="{policy}" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE}" -e AWS_DEFAULT_REGION="{region}" -e dry_run="yes" -e LDAP_HOST_NAME="{LDAP_HOST_NAME}" -e special_user_mails={special_user_mails} -e account_admin="{account_admin}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}"  -e policy_output="s3://{BUCKET_PERF_SCALE}/{LOGS}/{region}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
+    # dry_run="no" for perf-dept, perf-scale
```
Comment (athiruma) [inline jenkins/clouds/aws/policies/run_policies.py:90]: empty_buckets, empty_roles

Comment (ebattat) [reply jenkins/clouds/aws/policies/run_policies.py:90]: dry_run=yes for psap ?

Comment (athiruma) [reply jenkins/clouds/aws/policies/run_policies.py:90]: yes

File: jenkins/clouds/aws/policies/run_policies.py [hunk 3]
```diff
@@ -31,70 +31,59 @@
...
+            os.system(f"""podman run --rm --name cloud-governance --net="host -e account="PERF-SCALE" -e policy="{policy}" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE}" -e AWS_DEFAULT_REGION="{region}" -e dry_run="no" -e LDAP_HOST_NAME="{LDAP_HOST_NAME}" -e special_user_mails={special_user_mails} -e account_admin="{account_admin}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}"  -e policy_output="s3://{BUCKET_PERF_SCALE}/{LOGS}/{region}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
+        # running policies dry_run=yes per every region  ebs_unattached, ec2_stop, zombie_elastic_ips
+        elif policy not in ('empty_roles', 'empty_buckets'):
```
Comment (athiruma) [inline jenkins/clouds/aws/policies/run_policies.py:73]: @ebattat  this checks if we miss the above condition. If the above condition is false it sill executes empty_buckets.

Comment (ebattat) [reply jenkins/clouds/aws/policies/run_policies.py]: Looks ok

**Pull request merged**